notebooks/data-sources/TestGrid/metrics/unexpected_test_failures.ipynb

```diff
@@ -0,0 +1,973 @@
+{
```

Comment:
<p>How are you bringing the score into the 0-1 range? What does 0 mean and what does 1 mean?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/ocp-ci-analysis/pull/448/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='39'/>